bin/migrate-oats-data/migrate.py

```diff
@@ -56,6 +57,13 @@
                 application_submission_import(console, args)
             case "obfuscate":
                 start_obfuscation(console)
+            case "app-status-import":
+                application_status_import(console, args)
+            case "app-all-import":
+                application_import(console, args)
+                app_prep_import(console, args)
+                application_submission_import(console, args)
+                application_status_import(console, args)
```

Comment:
why do you need app-all-import? how is it different from app-import? Also, the migrate.py should not hold the logic of the sequential calling of functions. Migrate.py should only call the main function that abstracts all required child function calls.